examples/ecs/iter_combinations.rs

```diff
@@ -101,29 +108,34 @@ fn generate_bodies(
     }
 
     // add bigger "star" body in the center
+    let star_radius = 1.;
     commands
         .spawn_bundle(BodyBundle {
             pbr: PbrBundle {
-                transform: Transform {
-                    scale: Vec3::splat(0.5),
-                    ..Default::default()
-                },
+                transform: Transform::from_scale(Vec3::splat(star_radius)),
                 mesh: meshes.add(Mesh::from(shape::Icosphere {
                     radius: 1.0,
                     subdivisions: 5,
                 })),
-                material: materials.add((Color::ORANGE_RED * 10.0).into()),
+                material: materials.add(StandardMaterial {
+                    base_color: Color::ORANGE_RED,
+                    emissive: (Color::ORANGE_RED * 2.),
+                    ..Default::default()
+                }),
                 ..Default::default()
             },
-            mass: Mass(1000.0),
+            mass: Mass(500.0),
             ..Default::default()
         })
         .insert(PointLight {
-            color: Color::ORANGE_RED,
-            ..Default::default()
-        });
+            color: Color::WHITE,
+            intensity: 400.,
+            range: 100.,
+            radius: star_radius,
```

Comment:
```suggestion
            radius: star_radius,
            ..Default::default(),
```
This should fix the compile failure the CI caught.